File: lista_01/03/ContadorPar.java
import java.util.Scanner;

public class ContadorPar {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int totalNumeros = 10;
        int contaPares = 0;

        System.out.println("Digite 10 números inteiros:");

        for (int i = 1; i <= totalNumeros; i++) {
            System.out.print("Número " + i + ": ");
            int numero = scanner.nextInt();

            // Verifica se o número é par
            if (numero % 2 == 0) {
                contaPares++;
            }
        }

        System.out.println("Você digitou " + contaPares + " números pares.");
        scanner.close();
    }
}

File: lista_01/03/ContadorPar.py
def contar_numeros_pares():
    total_numeros = 10
    conta_pares = 0  
    print("Informe 10 números inteiros:")

    for i in range(1, total_numeros + 1):
        try:
            numero = int(input(f"Número {i}: "))
            # Verifica se o número é par
            if numero % 2 == 0:
                conta_pares += 1
        except ValueError:
            print("Entrada inválida. Insira um número inteiro.")
            i -= 1  # Decrementa o índice para repetir a entrada do número

    print(f"Você digitou {conta_pares} números pares.")

if __name__ == "__main__":
    contar_numeros_pares()
